babel.config.json

```diff
@@ -3,10 +3,14 @@
     "@babel/preset-env"
   ],
   "plugins": [
-    "@babel/plugin-proposal-class-properties",
-    "@babel/plugin-transform-object-assign"
+    "@babel/plugin-proposal-class-properties"
   ],
   "env": {
+    "umd": {
```

Comment:
good call on this. I had been thinking we should do this as well. does babel put extra junk in the es6 build because we use it on the static properties? if so then maybe we should just hand code the static fields onto the constructors or something

Reply:
Yeah, we should probably get rid of the proposal features and babel step.